docs/source/services/timelock_service/reverse-migration.rst

```diff
@@ -0,0 +1,140 @@
+.. _timelock-reverse-migration:
+
+Reverse Migration
+=================
+
+.. danger::
+
+   AtlasDB does not natively support reverse migrations. Improperly executing reverse migration from external timestamp
+   and lock services can result in **SEVERE DATA CORRUPTION**! Please contact the AtlasDB team before attempting a
+   reverse migration.
+
+If one wishes to downgrade from an external TimeLock Server to embedded timestamp and lock services, one can perform
+the inverse of the database migrations mentioned in :ref:`manual-timelock-migration`. It is also important to update the
+embedded timestamp bound to account for any timestamps issued since the original migration.
+
+The reverse migration process must be run offline (with no AtlasDB clients running) and consists of the following steps.
+Throughout this document, we assume the AtlasDB client you are trying to reverse-migrate is ``client`` .
+
+#. Shut down your AtlasDB clients.
+#. Take a restore timestamp.
+#. Update your key-value service to the restore timestamp.
+#. Revert your AtlasDB client configuration.
+#. Shut down the TimeLock servers.
+#. Remove your client from the list of TimeLock clients.
+#. Restart your AtlasDB clients.
+#. Restart the TimeLock servers.
```

Comment:
nit: restart -> start

If I was doing this migration, I'd do this:

1. Stop client
2. Alter timelock configuration
3. Restart timelock
4. Alter client config, start client

My motivation is that I probably have many services depending on timelock, and i don't want to have an unnecessary outage in them